Testing/Exec/Make.PelePhysics

```diff
@@ -71,7 +71,7 @@ ifeq ($(USE_F90_PP),TRUE)
     INCLUDE_LOCATIONS += $(REACTIONS_HOME_F90) $(REACTIONS_PATH_F90)
     VPATH_LOCATIONS   += $(REACTIONS_HOME_F90) $(REACTIONS_PATH_F90)
 endif
-ifdef Chemistry_Model
+#ifdef Chemistry_Model
```

Comment:
Seems weird to have to use defines in GNUmakefiles

Reply:
Yes, I commented them out in case but I'll remove them completely.